build/blueprints/template/files/src/patternfly/__typeDirectory__/__namePascalized__/examples/__namePascalized__.md

```diff
@@ -2,14 +2,43 @@
 title: '{{nameCapitalized}}'
 section: beta
 beta: true
-cssPrefix: pf-c-{{nameCapitalized}}
+cssPrefix: pf-c-{{nameDasherized}}
 ---
 
+## Optional introduction
+
+Optional introduction for `{{nameDasherized}}`.
+
 ## Examples
+
 ```hbs title=Basic
 {{moduleHbOpen}}
-  Basic example content
+  {{moduleHbIdConcatenationOpen}}
+    Item
+  {{moduleHbIdConcatenationClose}}
 {{moduleHbClose}}
 ```
 
 ## Documentation
+
+### Overview
+
+`{{nameDasherized}}` is a component. Some of its elements are presentational, while others require accessibility support.
+
+### Accessibility
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden. |
+| `aria-hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden from and unfocusable to screen readers, but remains visible for sighted users. |
+| `aria-label` | `.{{nameDasherized}}__item` | Provides an accessible name for a test element. **Required** |
+| `aria-expanded="true"` | `.{{nameDasherized}}__item` | Indicates that the expandable content is visible. **Required** |
+| `aria-expanded="true"` | `.{{nameDasherized}}__item` | Indicates that the expandable content is hidden. **Required** |
```

Comment:
do we need `aria-expanded`? It isn't as commonly used as the others.

Reply:
My thought here is include most of the the aria attributes w/descriptions that we use to drive consistency.

Reply:
Don't have to, just a thought that might make our lives easier w/copy / pasta

Reply:
Yeah I'm just thinking we'll end up removing it in most cases. I'm fine either way though. If you want to keep it just need to update the 2nd one to `aria-expanded="false"`

Reply:
We can expand on it ;D